resources.html

```diff
@@ -21,23 +21,30 @@ <h2 class="title">Editor</h2>
     </div>
 </div>
 <h2 class="title">Curriculum &amp; Lesson Plans</h2>
-<p>
-    In Progress - contact us for more info
-</p>
+<h3 class="title">AI/ML w/ North Hollywood Lesson Materials</h3>
```

Comment:
Could we try "Artificial Intelligence and Machine Learning at North Hollywood High School"? Not sure if the title is too unwieldly though!